connectors/github_mcp_connector.py

```
@@ -16,736 +16,730 @@
-                    
+
         except Exception as e:
             logger.error(f"Get rate limit failed: {e}")
-            return {
-                'success': False,
-                'error': str(e)
-            }
-    
+            return {"success": False, "error": str(e)}
+
     async def _update_rate_limit(self):
```

Ruff (0.11.9)

Missing return type annotation for private function `_update_rate_limit`

Add return type annotation: `None`

(ANN202)